.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +0,0 @@
-blank_issues_enabled: true
-contact_links:
-  - name: Feature or Enhancement Request
-    url: https://github.com/pieces-app/support/discussions
-    about: Post your idea in discussions for the Pieces team and community to discuss!
```

Comment:
I want to keep this config file so that people can still go to our support repo for product feature requests.

You can change the `blank_issues_enabled` boolean to false though since we won't need it

Reply:
Also, we should add to the `- name: ` so that it includes feature/enhancement requests as well as bug reports